sklearn/model_selection/_validation.py

```diff
@@ -207,6 +213,11 @@ def cross_validate(
                 The estimator objects for each cv split.
                 This is available only if ``return_estimator`` parameter
                 is set to ``True``.
+            ``indices``
+                The train/test indices for each cv split. A dictionary
+                is returned where the keys are either `"train"` or `"test"`
+                and the associated values are a list of NumPy arrays with the
+                indices. Available only if `return_indices=True`.
```

Comment:
Let's make it explicit that we do not attempt to use any pandas provided indices here.

```suggestion
                The train/test positional indices for each cv split. A dictionary
                is returned where the keys are either `"train"` or `"test"`
                and the associated values are a list of integer-dtyped NumPy
                arrays with the indices. Available only if `return_indices=True`.
```